import-secrets/handler.go

```diff
@@ -157,3 +153,10 @@ type SealedSecret struct {
 	Metadata   *metav1.ObjectMeta `yaml:"metadata"`
 	Spec       SealedSecretSpec   `yaml:"spec"`
 }
+
+func readBool(key string) bool {
```

Comment:
Can you reference this from the sdk package instead?

Reply:
I thought about that but the function is not in the sdk package so it will fail, maybe when this is merged and package bumped up we can make small pr referencing it from the SDK